symphony/bdk/core/activity/registry.py

```diff
@@ -47,10 +47,18 @@ def register(self, activity: AbstractActivity):
         :param activity: any object inheriting from base :class:`AbstractActivity`
         """
         logger.debug("Registering new activity %s", activity)
+        self._pre_process_activity(activity)
         self._activity_list.append(activity)
 
+    def _pre_process_activity(self, activity: AbstractActivity):
+        if isinstance(activity, SlashCommandActivity):
+            for act in self._activity_list:
+                if act.__eq__(activity):
```

Comment:
Pythonic way to call __eq__ is simply to write `activity == act`

Reply:
changed!